src/api.v2/model/inviteAccess.js

```diff
@@ -0,0 +1,19 @@
+const createGuts = require('../helpers/modelGuts');
+
+const tableName = 'invite_access';
+
+const selectableProps = [
+  'user_email',
+  'experiment_id',
+  'access_role',
+  'updated_at',
+];
+
+const guts = createGuts({
```

Comment:
this `createGuts` function is actually not implemented in `modelGuts` or anywhere else. How does this work then?